tests/test_selectize.py

```diff
@@ -0,0 +1,12 @@
+from HttpBase import HttpBase
+
+
+class TestSelectize(HttpBase):
+
+    def test_set_selectize(self):
```

Comment:
Started with a pretty simple test here. Let me know if you want coverage on any other cases.